repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
we'll want to distinguish this name from "resources that pods will consume". `resource-roles` doesn't do this for me.